### PR TITLE
Fixes #118 - Updated Design with Recommended UX Changes

### LINKED ISSUE #118: Update Plasmic from UX Review on 2022-02-11
- [ ] Variant where, Fix Learn More button, just make button the title 
- [ ] Make screenshot only in learn more
- [ ] Variant where add/remove is only on learn more page
- [ ] Modify description, remove gray background change the text
- [ ] Make search filter more prominent

### PR DESCRIPTION
**Pull request by danie1zhang**
- Updated Home.jsx to include the new Plasmic changes that were recommended during a UX review.
- Modified search filter to also filter based on description as well.

### CODE REVIEW

File: frontend/components/Home.jsx [hunk 1]
```diff
@@ -74,7 +77,7 @@ function Home__RenderFunc(props,ref) {
         }
         else {
             console.log("FILTERING DATA");
-            const filteredData = APIData.filter((item) => item.name.toLowerCase().includes(searchFilter.toLowerCase()))
+            const filteredData = APIData.filter((item) => (item.name.toLowerCase().includes(searchFilter.toLowerCase()) || item.short_description.toLowerCase().includes(searchFilter.toLowerCase())))
```
Comment (jonespm) [inline frontend/components/Home.jsx:80]: Cool quick fix for this

File: frontend/components/Home.jsx [hunk 2]
```diff
@@ -91,9 +94,12 @@ function Home__RenderFunc(props,ref) {
 					data-plasmic-for-node={forNode}
 					className={classNames(
 						"plasmic_default__all",
-						"plasmic_default__div",
-						"root_reset_mXv5TZ5SUPGRneH9RoMn6q",
-						"Home__root__rfWi8"
+                        "plasmic_default__div",
```
Comment (jonespm) [inline frontend/components/Home.jsx:97]: Looks like tabs changed to spaces here. But we can fix the formatting later

File: frontend/components/Home.jsx [hunk 3]
```diff
@@ -168,9 +174,7 @@ function Home__RenderFunc(props,ref) {
                                     "Home__text__k3LxN"
                                 )}
                             >
-                                {
-                                    "Canvas App Explorer is a collection of resources to assist the instructor in using the best tools available for you and your students. "
-                                }
+                                {"Find the Best Tools for Your Class and Students"}
```
Comment (jonespm) [inline frontend/components/Home.jsx:177]: We want to eventually turn this into a slot, I think we have an issue for that though.

**Review by jonespm — APPROVED** [review]

File: frontend/components/Home.jsx (hunk 2 above)
Comment (danie1zhang) [reply frontend/components/Home.jsx:97]: Oh yeah I think it's because I copy pasted it.

**Pull request merged**